### ISSUE TITLE
Automated scrolling while dragging and dropping

### ISSUE THREAD

**Issue by sakaggar29** [bevacqua/dragula#482]
While dragging a item below the size of window it is not providing automated scrolling. I have to use my mouse to scroll below the screen size. I am stuck with this from long time. How can this problem be resolved?

**Comment by shennan** [comment]
See [this issue]

**Comment by sakaggar29** [comment]
I went through it but didn't prove fruitful as it is still an open issue. Still facing the same problem. It's very generic and essential requirement which I feel is supposed to be taken care of by dragula. There are other libraries like sortable which deliver the same functionality.

**Comment by shennan** [comment]
@sakaggar29 Yes but it's the same issue, open or not, so it's best to show support for the current issue rather than start a new one.

**Comment by JuanJo4** [comment]
See #121 and comment [here](https://github.com/bevacqua/dragula/issues/121#issuecomment-180374772) where it suggest to use [dom_autoscroller](https://github.com/hollowdoor/dom_autoscroller)
